### PR TITLE
Build and install guide update

### PR DESCRIPTION
**Pull request by bettinaheim**
... based on feedback we received.

### CODE REVIEW

File: docs/sphinx/data_center_install.rst [hunk 1]
```diff
@@ -253,12 +268,19 @@ command to build CUDA Quantum:
     :start-after: [>CUDAQuantumBuild]
     :end-before: [<CUDAQuantumBuild]
 
-The CUDA Quantum build will compile or omit optional components automatically depending
-on whether the necessary pre-requisites are found in the build environment.
-Please check the build log to confirm that all desired components have been built. If
-you see a message that a component has been skipped, make sure you followed the 
-instructions for installing the necessary prerequisites and build dependencies, 
-and have set the necessary environment variables as described in this document.
+Please check the build log to confirm that all desired components have been built.
```
Comment (efratshabtai) [inline docs/sphinx/data_center_install.rst:271]: Where can they find the build log and what is its name?

**Review by efratshabtai — APPROVED** [review]

File: docs/sphinx/data_center_install.rst (hunk 1 above)
Comment (bettinaheim) [reply docs/sphinx/data_center_install.rst:271]: The build log is printed to the console when you execute the command above.

Comment (bettinaheim) [inline docs/sphinx/data_center_install.rst:271]:
```suggestion
Please check the build log printed to the console to confirm that all desired components have been built. 
```

Comment (bettinaheim) [reply docs/sphinx/data_center_install.rst]: rephrased

**Pull request merged**